SPEC-annex-B-elf-embedding.md

```diff
@@ -0,0 +1,29 @@
+# Annex B
+
+Annex B contains a method of embedding OmniBOR ID into elf files.
+
+### Embedded Artifact Dependency Graph Identifier
```

Comment:
Jump here from heading level 1 to heading level 3. Should either be a level 2 heading, or there should be a level 2 heading between them.

Reply:
Fixed